### PR TITLE
Create LICENSE for CMPUT401T03F13/#8

### PR DESCRIPTION
**Pull request by afontaine**
Using MIT license. Please add proper header to all files from now on.

### CODE REVIEW

**Comment by Jtfinlay** [comment]
Approved. Everyone else should approve this as well @jesseDtucker

**Comment by jviau** [comment]
Approved. We could use toilet paper for a license for all I care.

**Comment by Jtfinlay** [comment]
@edegraff - say that you're cool with this so we can merge. Else I'll throw a shoe.

**Comment by Jtfinlay** [comment]
@afontaine you spelled Evan's last name wrong btw

**Comment by edegraff** [comment]
I fear no shoe! But approved.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
+
+Copyright (c) 2013 Andrew Fontaine, James Finlay, Jesse Tucker, Jacob Viau, and Evan DeGraffe
```
Comment (Jtfinlay) [inline LICENSE]: *DeGraff

**Comment by jesseDtucker** [comment]
Approved

**Comment by afontaine** [comment]
Name fixed, pulling in.

**Pull request merged**